pkg/api/install/install.go

```diff
@@ -89,7 +89,9 @@ func enableVersions(externalVersions []unversioned.GroupVersion) error {
 }
 
 // userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
+var userResources = []string{"daemonsets", "deployments",
```

Comment:
`daemonsets` are privileged, they should be removed from the list.  Their pods have higher scheduling priority on kubelets and are never evicted.  That makes them significantly more powerful that any other user created pods.  They should only be used by cluster-admins.  Might leave a comment in the code explaining that so that others don't add them back.

The rest of the list looks fine.  At some point we should reconsider `pods`, since we'd recommend that people use a higher level construct to create them, but we don't have to do it in this pr.